test/e2e/run.test.ts

```diff
@@ -0,0 +1,63 @@
+/// <reference types="jest-playwright-preset" />
```

Comment:
How come this is needed?

Reply:
This is for enabling type checking by TS server. It tells the compiler the types for `page`, `browser` and other stuff that is injected by the preset.

Reply:
hmmm, interesting - I had seen this but never needed it, usually installing the @types/.... was enough - do you know why this is different?

Reply:
Didn't know, but [now I know](https://www.xspdf.com/resolution/53212492.html).

> so long as you have "moduleResolution": "node", in your tsconfig. json file. When you install typings with npm install @types/my-favorite-library, they are inserted by default into the @types/ folder. This is the default place for TypeScript to search for definition files.

If I understood correctly, for `jest-playwright-preset` the `moduleResolution` is not `node` and should not be changed for the test env. They offer options for the `reference types` [here](https://github.com/playwright-community/jest-playwright#usage-with-typescript).